components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -344,6 +344,26 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
     // enabling, make requests to enable them, then we repeat this in a loop
     // until we have no more services left, or we try too many times.
     this._appendLine(`Getting enabled services for project ${project}..`);
+    await request(
+      {
+        body: JSON.stringify(
+          {
+            Msg: 'Echo from web app?',
+          }
+        ),
+        headers: { 'content-type': 'application/json' },
+        method: 'PUT',
+        uri: this._configSpec.appAddress + '/healthz',
+      },
+      (error, response, body) => {
+        if (!error) {
+          const msg = JSON.parse(response.body).Reply;
+          this._appendLine('From Backend' + msg);
+        } else {
+          this._appendLine('Backend error: ' + response.statusCode);
```

Comment:
We should probably fail the deployment at this point, no? What's the point of continuing if we won't be able to pass on the token and create the service account key, etc?